development.ini

```diff
@@ -1,22 +1,49 @@
 [app:main]
-use = egg:config
-# Set this to reload templates automatically
+use = egg:database
 pyramid.reload_templates = true
+pyramid.includes =
+    pyramid_debugtoolbar
+    pyramid_tm
 
-# Remove pyramid_debugtoolbar before launch
-pyramid.includes = pyramid_debugtoolbar
+sqlalchemy.url = sqlite:///%(here)s/sqltutorial.sqlite
```

Comment:
We should be using a postgres database (not sqlite) in production (and development too). For now, make this url configurable so that we can easily update to a postgres database in a different pull request.

Reply:
From experience, I would suggest instead to make the INI file path configurable, and simply override that via volume mount in production. Otherwise, you will need to decode the INI in code to resolve additional settings/env-vars to plug individual parts of the INI configuration.

Reply:
- Database switched to postgres
- Database URI held in variable